src/ILCompiler/src/Program.cs

```diff
@@ -309,6 +311,20 @@ private int Run(string[] args)
                         compilationRoots.Add(new RawMainMethodRootProvider(entrypointModule));
                     }
                 }
+                else if (_nativeLib)
+                {
+                    EcmaModule module = null;
+                    // Get first (and only) module
```

Comment:
Is this limitation necessary? We could go over all of them and create a `ExportedMethodsRootProvider` for each.